raiden/exceptions.py

```diff
@@ -1,3 +1,16 @@
+"""
+What do you want from this file?
+
+1. I need to look up when to raise what.
+    Then read on the docstrings.
+2. I have to add a new exception.
+    Make sure you catch it somewhere.
+    Especially, if your new exception indicates bug in the Raiden codebase,
```

Comment:
This does not make sense to me.

What are you trying to say? Can you perhaps rephrase so that I understand?

Reply:
If you want to add a new exception, you have to make sure that you catch the new exception somewhere.  Especially, if ...

sounds better?

Reply:
Hehe the line I don't get is the "Especially if, ..." one.

Perhaps it's the Especially that puts me off. If it goes away it makes more sense.

Reply:
Now I'm getting it.  Rewriting...

Reply:
Please have another look at this.

Reply:
makes more sense now. Thank you